### ISSUE TITLE
training loss is stuck on high values

### ISSUE THREAD

**Issue by dagnichz** [wushidonguc/Two-stream-action-recognition-keras#8]
i'm trying to reproduce the results and train a temporal network based on the optical flow input you suggested. 
when I run your temporal_train.py code. the learning loss is not decreasing below 1.6.
validation accuracy is also stuck around 0.3.
can you share your learning curve and training final results
(since i'm assuming your reported results is with the test procedure you suggested, and i'm intrested in the results for a single optical flow stack classification).

**Comment by hadarsh7798** [comment]
@dagnichz  Same problem with me. I am trying to train the temporal network on my custom dataset. The loss and accuracy values just oscillate around 0.69. Were you able to resolve this issue?

**Comment by shabu19** [comment]
Hi @dagnichz and @hadarsh7798, 
I'm also trying to reproduce the results. When I run the temporal_train.py code my accuracy curve increases while val_accuracy stucks or fluctuates around 0.5. 
I ran it for maximum 200 epochs with batch size of 128. 
I also changed the dropout in dense layers to 0.5 and started with a learning rate of 0.0001 (temporal_train_model.py)

What are your observation?
Is this the train_accuracy reported in the paper and repository or the val_accuracy? Because i don't see any way getting val_accuracy near to 80%
